charms/katib-controller/reactive/katib_controller.py

```diff
@@ -2,11 +2,12 @@
 import os
 from pathlib import Path
 from subprocess import run
+from base64 import b64encode
 
 import yaml
 
 from charms import layer
-from charms.reactive import clear_flag, hook, set_flag, when, when_not
+from charms.reactive import clear_flag, hook, set_flag, when, when_not, hookenv
```

Comment:
probably want to use the actual charmhelper in case charms.reactive ever stops importing this (making it indirectly available)